src/gui/sdlmain.cpp

```diff
@@ -2338,12 +2338,20 @@ void GFX_SetMouseHint(const MouseHint hint_id)
 
 void GFX_CenterMouse()
 {
-	int current_width  = 0;
-	int current_height = 0;
+	int width  = 0;
+	int height = 0;
+
+#if defined(WIN32)
+	const auto window_canvas_size = get_canvas_size(sdl.desktop.type);
+
+	width  = window_canvas_size.w;
+	height = window_canvas_size.h;
+#else
+	SDL_GetWindowSize(sdl.window, &width, &height);
+#endif
 
 	assert(sdl.window);
```

Comment:
Please move the assert before ifdef, so that it is triggered before calling `SDL_GetWindowSize()`.

Reply:
OK, changed.